Add-ons/UmbracoForms/Developer/Forms-in-the-Database/index.md

```diff
@@ -41,3 +41,14 @@ Please be aware that enabling the persisting of Umbraco Forms in the database is
 :::
 
 When you save the file, the site will restart and run a migration step, migrating the files to the Umbraco database.
+
+## Migrating forms in files into a site
+
+If you have a v8 site running that stores Forms in the database but you have forms in file format you need migrated from a different site, you can force Forms to run the migration of those files again.
```

Comment:
Tried to shorten this sentence a bit, as I had a hard time wrapping my head around it.
Let me know if it still captures what you were trying to say!

```suggestion
If you have a Umbraco 8 site running that stores Forms in the database but you have the forms in file-format, you can force Forms to run the migration of those files again.
```